bundles/core/org.eclipse.smarthome.core/ESH-INF/config/i18n.xml

```diff
@@ -43,6 +43,14 @@
 			</description>
 			<advanced>true</advanced>
 		</parameter>
+		<parameter name="location" type="text" required="false">
+			<label>Location</label>
```

Comment:
didn't we want to introduce a dedicated context here? It should then also be added at https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#supported-contexts.